dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1868,6 +1868,9 @@ public Response getWorkflowZip(@ApiParam(hidden = true) @Parameter(hidden = true
         @ApiParam(value = "workflowVersionId", required = true) @PathParam("workflowVersionId") Long workflowVersionId) {
 
         Workflow workflow = workflowDAO.findById(workflowId);
+        if (workflow == null) {
+            throw new CustomWebApplicationException("could not find tool", HttpStatus.SC_NOT_FOUND);
```

Comment:
tool -> workflow.

Reply:
:+1:  will try to catch in next round-up